client/src/components/common/Button/Button.jsx

```
@@ -12,22 +44,137 @@
+
+			case 'none':
+				if (showNotifications) {
+					setNotification({
+						open: true,
+						message: `Button "${label}" clicked!`,
+						severity: 'info',
+					});
+				}
+				break;
+
+			case 'custom':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)